### ISSUE TITLE
Transparent vertical gap between sprites when scaled.

### ISSUE THREAD

**Issue by rkdrnf** [pixijs/pixi.js#5748]
### Expected Behavior
Sprites must be aligned seamlessly.

### Current Behavior
There are 1pixel gap between sprites.

![image](https://user-images.githubusercontent.com/1143159/58776115-e1a19e00-8603-11e9-844c-33413774a802.png)
(this image is blurred because I captured it again. actual image on screen is very crisp)
You can check those gaps between black outline and blue shape inside. there's transparent vertical gaps between those shapes which shows green background.

The stage is scaled as 2 and all positions of sprites are integers. I'm sure those sprites are aligned perfectly in their positions, and also SCALE_MODE is set to NEAREST right after pixi script is imported.

This only occurs in iOS safari / chrome and I never had this issue in windows / mac chrome and mac safari.

Those textures used in the image are packed with TexturePacker and used as texture atlas. It's remarkable that those are affected by texture packer settings such as padding between sprites.
With different setting, I saw that adjacent texture's pixels are blended into the edge of the sprite. (colors are not actually blended or interpolated. just pixels from adjacent images are rendered at the edge of sprite if I don't add padding between sprites)

below is the settings I initialize my pixi app
```
            antialias: false,
            autoResize: false,
            resolution: 1,
            roundPixels: true,
            sharedTicker: true
```

And those images rendered by tilerenderer do not have this issue. they align perfectly to each other.
only sprites rendered by PIXI.heaven.Sprite have this issue.
```
PIXI.settings.SCALE_MODE = PIXI.SCALE_MODES.NEAREST;
PIXI.tilemap.TileRenderer.SCALE_MODE = PIXI.SCALE_MODES.NEAREST;
```

It seems some sprites are rendered with offset of 1 pixel to the right, with the right most cropped 1pixel.

Any advice or solution would be appreciated!

### Environment
pixi -v: 4.8.2
- **Browser & Version**: Mobile safari 12.1.1 / chrome
- **OS & Version**: iOS 12.1.1

**Comment by rkdrnf** [comment]
It got fixed it by setting ```PIXI.settings.PRECISION = 'highp';```
But I still don't know if it's the right way to fix it.

**Comment by rkdrnf** [comment]
From the fact that it's fixed by changing precision, maybe those are caused by alignments in images packed in spritesheet. Packing images aligned to specific x, y offset might remove this issue even in mediump precision.

**Comment by rkdrnf** [comment]
But I still can’t understand why precision fixes it. Because I’m using sprite sheet of size 1024x2048 which is Power of 2

**Comment by ivanpopelyshev** [comment]
Because we had a bug that was fixed in 4.8.3:

https://github.com/pixijs/pixi.js/pull/5218

**Comment by ivanpopelyshev** [comment]
Btw, experiment that i referenced in that PR shows problem even on 512x512. even highp isn't enough to make perfect pixels like canvas2d context does.
